sympy/printing/tests/test_codeprinter.py

```diff
@@ -49,7 +49,7 @@ def emptyPrinter(self, obj):
     c = CrashingCodePrinter()
 
     # these should not silently succeed
-    with raises(NotImplementedError):
+    with raises(ValueError):
         c.doprint(ImmutableSparseMatrix(2, 2, {}))
-    with raises(NotImplementedError):
+    with raises(ValueError):
         c.doprint(MutableSparseMatrix(2, 2, {}))
```

Comment:
Would NotImplementedError be more appropriate here?

Reply:
Or perhaps actually a special exception class.

Reply:
I agree. I added a new Exception class.